chips/cc26xx/src/prcm.rs

```diff
@@ -145,4 +145,16 @@ impl Clock {
 
         prcm_commit();
     }
+
+    pub fn enable_trng() {
+        let regs: &PrcmRegisters = unsafe { &*PRCM_BASE };
+        regs.sec_dma_clk_run
+            .modify(SECDMAClockGate::TRNG_CLK_EN::SET);
+        regs.sec_dma_clk_sleep
+            .modify(SECDMAClockGate::TRNG_CLK_EN::SET);
+        regs.sec_dma_clk_deep_sleep
+            .modify(SECDMAClockGate::TRNG_CLK_EN::SET);
+
+        prcm_commit();
+    }
```

Comment:
Is this function used?

Reply:
Yes, [here](https://github.com/cpluss/tock/blob/a45b66519920e3f579908ad7a6f425c7f6d1ea6b/chips/cc26xx/src/trng.rs#L92) :)